src/components/Avatar/Avatar.test.js

```diff
@@ -0,0 +1,216 @@
+/* global describe, expect, it, shallow, mount, beforeEach, afterEach  */
+import React from 'react';
+import sinon from 'sinon';
+import Avatar from './Avatar';
+
+describe('Avatar', () => {
+    const sampleData = {
+        name: 'Matt Davies',
+        src: 'https://randomuser.me/api/portraits/lego/2.jpg',
+    };
+
+    const sandbox = sinon.sandbox.create();
```

Comment:
`sandbox` is now available in unit test utils, so you can:
`import { sandbox } from '../../../test/unit/utils';`
and you won't need the `beforeEach` and `afterEach`

Reply:
I intentionally didn't use your utility function, because I needed the `beforeEach` and `afterEach` to be defined within the `describe`.

If they were defined outside of the `describe` I ran into issues with my spies.  Particularly when I was stubbing console.error

Reply:
Arguably, we want to take advantage of `sandbox` as much as possible, but based on this, its API isn't ideal for most circumstances.

Reply:
@jfrej and I can take a look at this next :)